src/utilities/get.ts

```diff
@@ -0,0 +1,17 @@
+import {GeneralObject} from '../types';
+
+function get(obj: GeneralObject | undefined, keypath: string) {
+  if (obj == null) return undefined;
+
+  const keys = keypath.split('.');
```

Comment:
I'm also assuming that this is only checking for `a.b.c` format and not `['a', '0', 'b', 'c']` format?

Reply:
I noticed this also that it doesn't seem to support this use case but maybe that's because of what Solona said, it's not meant to be as robust. 
 `const object = { 'a': [{ 'b': { 'c': 3 } }] };
 get(object, 'a[0].b.c');
// => 3` 

// => actually returns undefined

Reply:
Comment above ^